README.md

```diff
@@ -50,6 +50,13 @@ The simplest way to install TensorLayer is:
 [for stable version] pip install tensorlayer
 ```
 
+Dockerfile is supplied to build images, build as usual
+
+```bash
+[for CPU version] docker image build -t tensorlayer:latest -f Dockerfile .
+[for GPU version] docker image build -t tensorlayer:latest-gpu -f Dockerfile.gpu .
```

Comment:
the commands can be simplified as 
```
docker build -t tensorlayer:latest -f Dockerfile .
docker build -t tensorlayer:latest-gpu -f Dockerfile.gpu .
```
The comment `[for CPU version] ` is not a valid bash code, so I also suggest rewrite them as valid bash comment, 
e.g. 
```
docker build -t tensorlayer:latest -f Dockerfile .          # for CPU version
docker build -t tensorlayer:latest-gpu -f Dockerfile.gpu .  # for GPU version
```
or 
```
# for CPU version
docker build -t tensorlayer:latest -f Dockerfile .
```

@luomai , @zsdonghao I just noticed that our example commands for `pip install` should also be rewritten as:
```
pip install tensorlayer  # for stable version
```

Reply:
The simplest way to install TensorLayer is: 

```bash
# for master version(Recommended)
$ pip install git+https://github.com/zsdonghao/tensorlayer.git 

# for stable version 
$ pip install tensorlayer
```

Dockerfile is supplied to build images, build as usual

```bash
# for CPU version
$ docker build -t tensorlayer:latest .

# for GPU version
$ docker build -t tensorlayer:latest-gpu -f Dockerfile.gpu . 
```

Reply:
add a space between here: `version (Recommended)`
LGTM!

Reply:
* Which command we should use?
```bash
pip install git+https://github.com/zsdonghao/tensorlayer.git
```
```bash
pip install git+https://github.com/tensorlayer/tensorlayer.git
```

Reply:
This one:
```
pip install git+https://github.com/tensorlayer/tensorlayer.git
```